web/src/components/apps/AppVersionHistory.jsx

```diff
@@ -80,13 +87,15 @@ class AppVersionHistory extends Component {
     currentPage: 0,
     pageSize: 20,
     loadingPage: false,
-  }
+    hasPreflightChecks: true,
+  };
 
   // moving this out of the state because new repeater instances were getting created
   // and it doesn't really affect the UI
-  versionDownloadStatusJobs = {}
+  versionDownloadStatusJobs = {};
 
   componentDidMount() {
+    this.getPreflightState(this.props.app.downstream.currentVersion);
     const urlParams = new URLSearchParams(window.location.search);
```

Comment:
run getPreflightState on mount to check if this version's preflightstate response will be empty